h/static/scripts/tests/controllers/search-bar-controller-test.js

```diff
@@ -83,6 +134,61 @@ describe('SearchBarController', function () {
         });
     });
 
+    describe('it allows tag value suggestions', function () {
```

Comment:
This is easier to follow and extend now that the individual test cases have been split up 👍